api/app/helpers/sendAlert.ts

```diff
@@ -7,51 +7,45 @@ function transformUrl(transcriptUrl?: string | null): string | null {
   }
 
   // Extract necessary parts from the transcriptUrl
-  const url = new URL(transcriptUrl);
-  const pathParts = url.pathname.split("/");
-  const transcriptPath = pathParts.slice(5).join("/");
+  const { pathname } = new URL(transcriptUrl);
+  const transcriptPath = pathname.split("/").slice(5).join("/");
 
-  // Construct the correct URL
-  const normalUrl = `https://btctranscripts.com/${transcriptPath}`;
+  // Construct the correct URL using template literals
+  const transcriptUrlPath = `https://btctranscripts.com/${transcriptPath}`;
 
-  return normalUrl;
+  // Remove .md from the end of the URL
+  const transformedUrlPath = transcriptUrlPath.replace(/\.md$/, "");
+
+  return transformedUrlPath;
 }
 
 export async function sendAlert(
   message: string,
+  isError: boolean = false,
   transcriptTitle?: string | null,
   speakers?: string | null,
   transcriptUrl?: string | null,
-  isError?: boolean
 ) {
   //bypass alerts in development
   if (process.env.NODE_ENV === "development") {
     return;
   }
 
-  let webhookUrl;
-  if (isError) {
-    webhookUrl = process.env.DISCORD_ERROR_WEBHOOK_URL;
-  } else {
-    webhookUrl = process.env.DISCORD_TRANSCRIPT_WEBHOOK_URL;
-  }
+  // Use ternary operator for setting webhookUrl
+  const webhookUrl = isError ? process.env.DISCORD_ERROR_WEBHOOK_URL : process.env.DISCORD_TRANSCRIPT_WEBHOOK_URL;
 
   if (!webhookUrl) {
     throw new Error("Webhook URL is not set");
   }
 
   const transformedUrl = transformUrl(transcriptUrl);
 
-  let content;
-  if (isError) {
-    content = `Error: ${message}`;
-  } else {
-    content = `🤖 New Transcript Ready for Review!\nTitle: ${transcriptTitle}\nSpeakers: ${speakers}\nLink: ${transformedUrl}`;
-  }
+  // Use template literals for constructing content
+  const content = isError ? `Error: ${message}` : `🤖 ${message}\nTitle: ${transcriptTitle}\nSpeakers: ${speakers}\nLink: ${transformedUrl}`;
```

Comment:
let's add a message title for when we have an error. Maybe:
"❌ Transcript not added to the Queue"